### PR TITLE
Update Package.swift to use upToNextMajor

### PR DESCRIPTION
**Pull request by kevinlind**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,8 +61,8 @@ dependencies: [
 	.package(url: "https://github.com/adobe/aepsdk-edge-ios.git", from: "1.1.0"),
```
Comment (sbenedicadb) [inline README.md:61]: we should update this to use `.upToNextMajor(from: "1.1.0")`

File: README.md [hunk 2]
```diff
@@ -61,8 +61,8 @@ dependencies: [
...
 targets: [
```
Comment (sbenedicadb) [inline README.md:62]: i think the `targets` array and `dependencies` array are meant to be on the same level of this block - currently `targets` is nested inside of `dependencies`, and it should be moved out

**Review by sbenedicadb — CHANGES_REQUESTED** [review]

**Comment by sbenedicadb** [comment]
@kevinlind can we just merge this into main directly since there are unreleased changes on `dev`?

**Comment by kevinlind** [comment]
@sbenedicadb Yes, good catch. Let me rebase my changes from main.

**Comment by kevinlind** [comment]
@sbenedicadb As the rebase changes the commit history from what is in this PR, I'll need to close and create a new one.

**Pull request closed without merging**